### PR TITLE
bat: new package at 0.15.4

### PR DESCRIPTION
**Pull request by git-bruh**
## Description of package
A `cat` clone with useful features like syntax highlighting and `git` integration.

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/bat/version
/var/db/kiss/installed/bat/sources
/var/db/kiss/installed/bat/post-install
/var/db/kiss/installed/bat/manifest
/var/db/kiss/installed/bat/depends
/var/db/kiss/installed/bat/checksums
/var/db/kiss/installed/bat/build
/var/db/kiss/installed/bat/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/bat.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/bat
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/bat/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+cargo build --release
+
+install -Dm755 target/release/bat "$1/usr/bin/bat"
+find . -name bat.1 -exec install -Dm644 {} \
```
Comment (dylanaraps) [inline community/bat/build:6]: Do we not know where the file is?

Comment (git-bruh) [reply community/bat/build:6]: The man-page is generated at a path that looks like this:
`./target/release/build/bat-484c9d3dc1d6eff0/out/assets/manual/bat.1`
If i compile the git version then the path is like this:
`./target/release/build/bat-ef405f4526930eea/out/assets/manual/bat.1`
Hence, `find` is used as the man-page path will probably change over new releases

Comment (dylanaraps) [reply community/bat/build:6]:
```sh
install -Dm644 target/release/build/bat-*/out/assets/manual/bat.1 "$1/usr/share/man/man1"
```

Does this work?

Comment (dylanaraps) [reply community/bat/build:6]: Can we not install everything using `cargo`?

Comment (git-bruh) [reply community/bat/build]: Cargo doesn't seem to install the man-page, only the executable is installed.
I've added a commit to use the method you suggested for installing the man-page.

Comment (git-bruh) [reply community/bat/build]: Seems like cargo doesn't yet support installing man-pages either, so it'll have to be done manually. See https://github.com/rust-lang/cargo/issues/2729

**Pull request merged**